### DISCUSSION TITLE
Window Source Capture Support

### DISCUSSION THREAD

**Discussion by SiggeVibes** [geovens/gInk#215] [Ideas]
![image](https://github.com/user-attachments/assets/80884b40-7056-447b-ada3-60cce398aea6)

It would be nice if it would be possible to add this so in OBS you could record gInk as a specific window rather than having to record the whole screen which I would like to avoid.

**Comment by pubpub-zz** [comment]
you should have a look here:
https://github.com/pubpub-zz/ppInk/issues/101
